pokt-network/banano-quest
# Leaderboard screen implementation

![](https://media.giphy.com/media/9JxRQ6NOf1orK/giphy.gif)

What's changed?
Added two operations for getting the ownersCount and the owner'sTokenCount.
Added another storyboard and TableViewCell class for the cell inside the screen.
The leaderboard is first COMPLETELY loaded before presenting the records because the sorting happens after all the records have been retrieved.

How to test
1- Log into the app and tap on leaderboard in the top left menu.
2- The leaderboard should load after a few seconds
